dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/merger/MergerFactory.java

```diff
@@ -49,6 +53,11 @@
                 result = mergerCache.get(returnType);
             }
         }
+
+        if (result == null) {
```

Comment:
I don't think we should throw an exception, let's just return null.

Reply:
May be we can add some comments about this NullPointerException

Reply:
I still think it is ok to return null.